microservices/is_eligible_microservice.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+from perun.connector.utils.Logger import Logger
+from satosa.context import Context
+from satosa.internal import InternalData
+from satosa.micro_services.base import ResponseMicroService
+from satosa.response import Redirect
+
+from utils.PerunConstants import PerunConstants
+
+logger = Logger.get_logger(__name__)
+
+
+class IsEligible(ResponseMicroService):
+    def __init__(self, config, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        logger.info("is active")
+
+        self.__TRIGGER_ATTRIBUTE = "trigger_attribute"
+        self.__ELIGIBLE_LAST_SEEN_TIMESTAMP_ATTRIBUTE = (
+            "eligible_last_seen_timestamp_attribute"
+        )
+        self.__VALIDITY_PERIOD_MONTHS = "validity_period_months"
+        self.__DEFAULT_VALIDITY_PERIOD_MONTHS = 12
+
+        self.__filter_config = config["filter_config"]
+
+        self.__trigger_attribute = self.__filter_config[
+            self.__TRIGGER_ATTRIBUTE
+        ]
+        self.__timestamp_attribute = self.__filter_config[
+            self.__ELIGIBLE_LAST_SEEN_TIMESTAMP_ATTRIBUTE
+        ]
+        self.__validity_period_months = self.__filter_config.get(
+            self.__VALIDITY_PERIOD_MONTHS,
+            self.__DEFAULT_VALIDITY_PERIOD_MONTHS,
+        )
```

Comment:
this variable (default) is not needed as a member of `self`, so either move it outside the class, or even better:
```suggestion
        self.__validity_period_months = 12

        self.__filter_config = config["filter_config"]

        self.__trigger_attribute = self.__filter_config[
            self.__TRIGGER_ATTRIBUTE
        ]
        self.__timestamp_attribute = self.__filter_config[
            self.__ELIGIBLE_LAST_SEEN_TIMESTAMP_ATTRIBUTE
        ]
        self.__validity_period_months = self.__filter_config.get(
            self.__VALIDITY_PERIOD_MONTHS,
            self.__validity_period_months,
        )
```